tools/tests/test_nd_freeze

```diff
@@ -63,7 +60,7 @@ function test_setup {
 # Run commands that are run at the end of each test run.
 function test_teardown {
     echo "INFO: removing TMP_DIR: ${TMP_DIR}"
-    rm -rf $TMP_DIR
+    # rm -rf $TMP_DIR
```

Comment:
I thought I left a comment but do not longer see it here -- why commented out? (also mention that `echo` above still says that things do get removed)

Reply:
That comment-out should not have been committed. I need to look at my diffs more closely before I commit

Reply:
> I thought I left a comment but do not longer see it here

You did. https://github.com/neurodebian/neurodebian/commit/097ad63bf5a8606dca1421c19f325fa6481f9f05#r33217475 

Maybe it's not showing up because it was somehow associating with the commit but not the PR (even though that is the same commit in this pull request)?